stackslib/src/chainstate/stacks/mod.rs

```diff
@@ -659,22 +675,73 @@ pub enum TenureChangeError {
 /// Schnorr threshold signature using types from `wsts`
 #[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
 pub struct ThresholdSignature(pub wsts::common::Signature);
+impl FromSql for ThresholdSignature {
+    fn column_result(value: ValueRef) -> FromSqlResult<ThresholdSignature> {
+        let hex_str = value.as_str()?;
+        let bytes = hex_bytes(&hex_str).map_err(|_| FromSqlError::InvalidType)?;
+        let ts = ThresholdSignature::consensus_deserialize(&mut &bytes[..])
+            .map_err(|_| FromSqlError::InvalidType)?;
+        Ok(ts)
+    }
+}
+
+impl ToSql for ThresholdSignature {
+    fn to_sql(&self) -> rusqlite::Result<ToSqlOutput> {
+        let bytes = self.serialize_to_vec();
+        let hex_str = to_hex(&bytes);
+        Ok(hex_str.into())
+    }
+}
 
 /// A transaction from Stackers to signal new mining tenure
 #[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
 pub struct TenureChangePayload {
+    /// Consensus hash of this tenure.  Corresponds to the sortition in which the miner of this
+    /// block was chosen.  It may be the case that this miner's tenure gets _extended_ across
+    /// subsequent sortitions; if this happens, then this `consensus_hash` value _remains the same_
+    /// as the sortition in which the winning block-commit was mined.
+    pub tenure_consensus_hash: ConsensusHash,
+    /// Consensus hash of the previous tenure.  Corresponds to the sortition of the previous
+    /// winning block-commit.
+    pub prev_tenure_consensus_hash: ConsensusHash,
+    /// Current consensus hash on the underlying burnchain.  Corresponds to the last-seen
+    /// sortition.
+    pub sortition_consensus_hash: ConsensusHash,
```

Comment:
Can this be called `burn_view` (or something like that) instead?

`sortition` implies that a sortition  occurred(i.e., cryptographic random-weighted leader selection), but unless I'm confused, this field is used specifically to deal with the case that sortition _didn't_ occur: a burn block was processed which didn't select a new miner. This is how the sortition_db uses the term, and I think it would be very confusing to use it differently.

Reply:
Sure, we can call it `burn_view`